commands/store_deploy.go

```diff
@@ -21,6 +22,7 @@ func init() {
 	storeDeployCmd.Flags().BoolVar(&storeDeployFlags.update, "update", true, "Update existing functions")
 	storeDeployCmd.Flags().StringArrayVar(&storeDeployFlags.constraints, "constraint", []string{}, "Apply a constraint to the function")
 	storeDeployCmd.Flags().StringArrayVar(&storeDeployFlags.secrets, "secret", []string{}, "Give the function access to a secure secret")
+	storeDeployCmd.Flags().BoolVarP(&storeDeployFlags.sendRegistryAuth, "send-registry-auth", "a", false, "send registryAuth from Docker credentials manager with the request")
```

Comment:
This shouldn't be needed at all. The function store are all public images. Adding it just adds unnecessary code to the store that has a potential for a security issue sending the credentials to the gateway when it's not needed.